Drv/Ip/UdpSocket.cpp

```diff
@@ -111,13 +105,9 @@ SocketIpStatus UdpSocket::bind(NATIVE_INT_TYPE fd) {
     if (::getsockname(fd, reinterpret_cast<struct sockaddr *>(&address), &size) == -1) {
         return SOCK_FAILED_TO_READ_BACK_PORT;
     }
-    U16 port = ntohs(address.sin_port);
 
-    this->m_lock.lock();
     FW_ASSERT(sizeof(this->m_state->m_addr_recv) == sizeof(address), sizeof(this->m_state->m_addr_recv), sizeof(address));
     memcpy(&this->m_state->m_addr_recv, &address, sizeof(this->m_state->m_addr_recv));
-    this->m_recv_port = port;
```

Comment:
Why scrub this line?

Reply:
This and the previous question is a really good question. I think I was trying to get rid of the local and messed up. Thank you!

Reply:
I'm kind of not sure why thinks worked with this issue. Seems like m_recv_port wouldn't have been assigned

Reply:
So actually, why does m_recv_port need to get assigned in `configureRecv` and `bind`?

Reply:
You are right, it is assigned at the top, and these two statements are superfluous.